dbt_platform_helper/providers/platform_config_schema.py

```diff
@@ -434,6 +436,21 @@ def valid_s3_bucket_name(name: str):
 
         return True
 
+    @staticmethod
+    def __datadog_schema() -> dict:
+        return {
+            "type": "datadog",
+            Optional("environments"): {
+                PlatformConfigSchema.__valid_environment_name(): {
+                    Optional("team_name"): str,
+                    Optional("contact_name"): str,
+                    Optional("contact_email"): str,
+                    Optional("documentation_url"): str,
+                    Optional("services_to_monitor"): list,
+                }
```

Comment:
Are the properties really optional?

Can we do something like this?

```suggestion
                Optional(PlatformConfigSchema.__valid_environment_name()): {
                    "team_name": str,
                    "contact_name": str,
                    "contact_email": str,
                    "documentation_url": str,
                    "services_to_monitor": list,
                }
```

This might apply elsewhere, but it would be good to see if it is possible with this one.

Really, we need at least "*" then other environments are optional, but I think that can wait for the outcome of [DBTP-1708 Should we use Pydantic for schema validation?](https://uktrade.atlassian.net/browse/DBTP-1708).

Reply:
@chiaramapellimt This was resolved without explanation. What's the thinking?